test/localDate.spec.js

```diff
@@ -59,3 +59,10 @@ test('If iso 8601 date string and timezone info, LocalDate uses new Date(dateStr
 
   expect(date.getTime()).toBe(nativeDate.getTime());
 });
+
+test('If milliseconds with dateString, LocalDate can get a valid date', () => {
+  const nativeDate = new Date('2020-01-29T19:20:00.999');
+  const date = new LocalDate('2020-01-29T19:20:00.999');
+
+  expect(date.getTime()).toBe(nativeDate.getTime());
+});
```

Comment:
How about adding all dateString types to TC?
```
2015-07-31T23:59:59.999
2015-07-31T23:59:59
2015-07-31T23:59:59.999+09:00
2015-07-31T23:59:59.999Z
2015-07-31T23:59:59+09:00
```

Reply:
OK